blog/2025-10-20-release-1.10.4.md

```diff
@@ -0,0 +1,146 @@
+
+See [PR 6828](https://github.com/kubevela/kubevela/pull/6828) or the [revised documentation](https://kubevela.io/docs/next/platform-engineers/status/definition_health_status) for more information.
+
+<!--truncate-->
+
+## Application Status Metrics
+As part of the fix in [PR 6824](https://github.com/kubevela/kubevela/pull/6824) we now ensures that Application health is continuously evaluated, even after the workflow succeeds.
+
+To compliment this, we've added additional application level metrics so that operators can have simplified visibility into the status of their Applications.  
+
+These are the:
+- `kubevela_application_health_status{app_name, namespace}`
+- `kubevela_application_phase{app_name, namespace}`
+- `kubevela_application_workflow_phase{app_name, namespace}`
+
+We've made efforts to ensure these are low cardinality through numeric phase mapping - so operators should be able to collect and monitor these cost effectively in their monitoring solutions. 
+
+To add more depth to monitoring capabilities, we've also added structured logging when Application changes occur. These logs capture the Application state in a high detail of detail to supplement the new metrics.
+
+The full structure of these logs can be found [here](https://kubevela.io/docs/next/platform-engineers/status/application_health_status_metrics/#structured-logging)
+
+To get started, users can enable the EnableApplicationStatusMetrics feature gate to activate both metrics and the new structured logging.
+
+See [PR 6857](https://github.com/kubevela/kubevela/pull/6857) or the [documentation](https://kubevela.io/docs/next/platform-engineers/status/application_health_status_metrics/) for more information.
+
+<!--truncate-->
+
+## Dependency-Aware Workflows
+[Issue 6852](https://github.com/kubevela/kubevela/issues/6852) raised the issue with the confusion of DependsOn [within components](https://kubevela.io/docs/end-user/workflow/component-dependency-parameter/#dependency), and DependsOn [within workflows](https://kubevela.io/docs/end-user/workflow/dependency/#how-to-use) - and how workflows did not natively respect component dependencies. 
+
+KubeVela will now dynamically add workflow dependencies between steps according to the components dependencies; which is vastly simplified and much more in line with user expectations. 
+
+See [PR 6854](https://github.com/kubevela/kubevela/pull/6854) for more information. 
+
+<!--truncate-->
+
+## Colorised Logging Support
+We've introduced opt-in support for colorised logging to improve the developer debugging experience. 
+
+For production and CI, this is disabled and existing log functionality is preserved. Developers can use the new `--dev-logs` to enable this for their console output.
```

Comment:
Didnt Vishal say that the colorized logs is only for the vscode logging. I am not sure how that was built, but would be good to confirm that piece